OpenRA.Mods.Common/Traits/SupportPowers/SupportPower.cs

```diff
@@ -20,6 +20,10 @@ public abstract class SupportPowerInfo : ITraitInfo
 		public readonly string LongDesc = "";
 		public readonly bool AllowMultiple = false;
 		public readonly bool OneShot = false;
+
+		[Desc("If set to true, the support power will be fully charged when it becomes available." +
+			"Normal rules apply for subsequent charges.")]
```

Comment:
I believe concatenating will lead to a missing space after the sentence end dot.